webapp/cypress/integration/auth.feature

```diff
@@ -0,0 +1,165 @@
+    And I enter "Pa$$word123@" into "Repeat your password" field on the "Register" Page
+    And I click on "Register" button on the "Register" Page
+    Then Alert "You have been successfully registered. You can now log in!" should be displayed on "Register" Page
+    And I should be reddirected to "Login" Page after click "Ok" button from alert
+
+  Scenario: Alert should be displayed when user with that email already exist
+    Given I am on the site homepage
+    When I click on "Register" button on the "Home" Page
+    And I enter "admin@admin.pl" into "Email" field on the "Register" Page
+    And I enter "Test User" into "Full name" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Password" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Repeat your password" field on the "Register" Page
+    And I click on "Register" button on the "Register" Page
+    Then Alert "Given email is already in use" should be displayed under "Email" field on the "Register" Page
+
+  Scenario: Alert should be displayed when I enter not valid email
+    Given I am on the site homepage
+    When I click on "Register" button on the "Home" Page
+    And I enter "test123.pl" into "Email" field on the "Register" Page
+    And I enter "Test User" into "Full name" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Password" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Repeat your password" field on the "Register" Page
+    And I click on "Register" button on the "Register" Page
+    Then Alert "Email not valid!" should be displayed under "Email" field on the "Register" Page
+
+  Scenario: Alert should be displayed when I left email form empty
+    Given I am on the site homepage
+    When I click on "Register" button on the "Home" Page
+    And I left empty "Email" field on the "Register" Page
+    And I enter "Test User" into "Full name" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Password" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Repeat your password" field on the "Register" Page
+    And I click on "Register" button on the "Register" Page
+    Then Alert "Email is required!" should be displayed under "Email" field on the "Register" Page
+
+  Scenario: Alert should be displayed when I left full name form empty
+    Given I am on the site homepage
+    When I click on "Register" button on the "Home" Page
+    And I enter "test@test.pl" into "Email" field on the "Register" Page
+    And I left empty "Full name" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Password" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Repeat your password" field on the "Register" Page
+    And I click on "Register" button on the "Register" Page
+    Then Alert "Full name is required!" should be displayed under "Full name" field on the "Register" Page
+
+    #zapytać sie o walidacje full name, bo mozna wpisac np Name@123
```

Comment:
nie można, dodałem taska żeby to naprawić LP-163.

Reply:
ok :+1: